i18n/qqq.json

```diff
@@ -22,7 +22,7 @@
 	"calculate-totals": "Label for button to calculate the statistics for an event.",
 	"cancel": "Label for button\n{{Identical|Cancel}}",
 	"categories": "Label for section showing MediaWiki categories associated with an Event.\n{{Identical|Category}}",
-	"category-description": "Description of how categories are used to generate statistics.",
+	"categories-description": "Description of how categories are used to generate statistics.",
```

Comment:
qqq for `categories-commons-explanation`?

Reply:
Done.